prow/manifests/overlays/metal3/config.yaml

```diff
@@ -1149,6 +1149,16 @@ presubmits:
       always_run: false
       # Until we have checked that it works, keep it optional
       optional: true
+    # name: {job_prefix}-{proj}-{capm3_target_branch}-e2e-basic-test-{image_os}
+    - name: metal3-project_infra-main-e2e-basic-test-centos
+      branches:
+        - main
+        - release-1.6
```

Comment:
release-1.5?

Reply:
as talked offline we are still unsure and we need to test the branches listed here first.